.circleci/config.yml

```diff
@@ -357,8 +357,10 @@ jobs:
       - run: diff-cover --compare-branch $(git rev-parse --abbrev-ref origin/HEAD) coverage.xml
 
   build_base_venvs:
-    executor: ddtrace_dev
-    parallelism: 6
+    resource_class: large
+    docker:
+      - image: *ddtrace_dev_image
+    parallelism: 7
```

Comment:
I wonder if we can get a default value/anchor for this defined in one place. If we ever need 8, or down to 5 or w/e